orchestrate.py

```diff
@@ -248,7 +302,8 @@ def main():
         (r"/(user/\w+)(?:/.*)?", LoadingHandler),
         (r"/api/stats/?", APIStatsHandler),
         (r"/stats/?", RedirectHandler, {"url": "/api/stats"}),
-        (r"/info/?", InfoHandler)
+        (r"/info/?", InfoHandler),
+        (r"/api/pool/1?", APIPoolHandler),
```

Comment:
I just realized this should be `/api/pools`. There's only ever one pool here, to be used by some higher level "make a new tmpnb/pool" via this API proposal: https://github.com/jupyter/enhancement-proposals/pull/5